cms/tests/test_compatibility_with_4x_branch.py

```diff
@@ -0,0 +1,24 @@
+import cms
```

Comment:
This isn't being used according to the checks so can be removed 👌

Reply:
thanks! (and done) :)